packages/labs/context/src/lib/value-notifier.ts

```diff
@@ -11,6 +11,16 @@ import {ContextCallback} from './context-request-event.js';
  */
 type Disposer = () => void;
 
+interface CallbackInfo {
+  disposer: Disposer;
+  consumerHost?: Element;
+}
+
+export interface MovedSubscription<T> {
```

Comment:
Nit: Could this exported interface be documented?

Reply:
Removed